src/commands/package.yml

```diff
@@ -0,0 +1,20 @@
+description: >
+  This command packaging the test project.
```

Comment:
I don't think we should include the Testlio package in the 1st version of the Orb. I'd rather keep it simple: the orb just cares about a file path to a zip file with test scripts. That zip package must follow the conventions we describe in our documentation.

At the moment this package command is very much tailored to our own use case and we do not know how other customers will configure their pipelines.